spec/form_models/jobseekers/account_feedback_form_spec.rb

```diff
@@ -3,4 +3,25 @@
 RSpec.describe Jobseekers::AccountFeedbackForm, type: :model do
   it { is_expected.to validate_inclusion_of(:rating).in_array(Feedback.ratings.keys) }
   it { is_expected.to validate_length_of(:comment).is_at_most(1200) }
+  it { is_expected.to allow_value("email@example").for(:email) }
```

Comment:
I'd put all the checks on email fields together on line 17, where validation is happening. same for the other form specs